flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -218,6 +222,11 @@ public RocksDBKeyedStateBackendBuilder(
 		return this;
 	}
 
+	RocksDBKeyedStateBackendBuilder<K> setColumnFamilyAsVariable(boolean isVariable) {
```

Comment:
We can make this part of the `RocksDBNativeMetricOptions` which should simplify the PR quite a bit.

Reply:
I get your idea, I will edit again.